test/manipulation.test.js

```diff
@@ -542,6 +543,22 @@ describe('manipulation', function() {
       });
     });
 
+    it('should reject created StubUser with empty password', function(done) {
+      StubUser.create({email: 'b@example.com', password: ''}, function(err, createdUser) {
+        (err.message).should.match(/password cannot be empty/);
+        done();
+      });
+    });
+
+    it('should reject updated empty password with updateAttribute', function(done) {
+      StubUser.create({password: 'abc123'}, function(err, createdUser) {
```

Comment:
in the `2.x` PR, the json contains the email as well.  Do we still need it? 
`StubUser.create({email: 'b@example.com', password: 'abc123'}, `